portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,13 +46,34 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
     response.setContentType("application/json;");
     response.getWriter().println(json);
-    response.sendRedirect("/index.html");
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("Comment:");
-    comments.add(comment);
+    long timeStamp = System.currentTimeMillis();
```

Comment:
`timestamp` is an ambiguous variable / property name for a long.

Right now a reader away from this definition will only know that it's a long representing a time and nothing else. If could very well be seconds, microseconds, nanoseconds, etc.

Consider changing to something like `timeMillis` or `submissionTimeMillis`.

P.S. if you already have comments in Datastore with the other property name you may need to wipe them.